sdk/src/main/java/io/radar/sdk/RadarActivityLifecycleCallbacks.kt

```diff
@@ -44,23 +45,38 @@ internal class RadarActivityLifecycleCallbacks(
     }
 
     override fun onActivityResumed(activity: Activity) {
-        if (count == 0) {
+        Log.w(TAG, "ON RESUME ${count} ${isFirstCreate}")
+        if (count == 0 && !isFirstCreate) {
             try {
                 val updated = RadarSettings.updateSessionId(activity.applicationContext)
                 if (updated) {
                     val usage = "resume"
                     Radar.apiClient.getConfig(usage, false, object : RadarApiClient.RadarGetConfigApiCallback {
                         override fun onComplete(status: Radar.RadarStatus, config: RadarConfig) {
-                            Radar.locationManager.updateTrackingFromMeta(config.meta)
-                            RadarSettings.setFeatureSettings(activity.applicationContext, config.meta.featureSettings)
+                            if (status == Radar.RadarStatus.SUCCESS) {
+                                Radar.locationManager.updateTrackingFromMeta(config.meta)
+                                RadarSettings.setFeatureSettings(activity.applicationContext, config.meta.featureSettings)
+                                RadarSettings.setSdkConfiguration(activity.applicationContext, config.meta.sdkConfiguration)
```

Comment:
again, not entirely sure why we need the set then get here.

Reply:
config.meta.sdkConfiguration might be null, in that case, we retrieve the value from storage